app/deliver_grant_funding/admin/views.py

```diff
@@ -524,27 +524,29 @@ def download_data_providers_csv(self, grant_id: UUID, collection_id: UUID) -> An
         csv_writer.writeheader()
 
         grant_recipients = get_grant_recipients(grant=grant, with_data_providers=True)
-        grant_recipient_data_providers = [
-            data_provider for grant_recipient in grant_recipients for data_provider in grant_recipient.data_providers
-        ]
-        for data_provider in sorted(grant_recipient_data_providers, key=lambda u: u.email):
-            reporting_period = format_date_range(
-                collection.reporting_period_start_date, collection.reporting_period_end_date
-            )
-            report_deadline = format_date(collection.submission_period_end_date)
-
-            # TODO: Update this to link directly to the grant recipient's report page once that route is available
-            grant_report_url = url_for("auth.request_a_link_to_sign_in", _external=True)
-
-            csv_writer.writerow(
-                {
-                    "email_address": data_provider.email,
-                    "grant_name": grant.name,
-                    "reporting_period": reporting_period,
-                    "report_deadline": report_deadline,
-                    "grant_report_url": grant_report_url,
-                }
-            )
+        for grant_recipient in grant_recipients:
```

Comment:
probably nice to sort grant recipients by org name? i've been doing that elsewhere; again one we could probably put on the relationship for consistency